pkgs/development/python-modules/adb-shell/default.nix

```diff
@@ -0,0 +1,25 @@
+{ aiofiles, buildPythonPackage, cryptography, fetchPypi, lib, libusb1, pyasn1
+, rsa }:
+
+buildPythonPackage rec {
+  pname = "adb-shell";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "adb_shell";
+    sha256 = "0bb9589jv4h38mv3rhxxpmnyc4xvm29brfx3gzgnn51w3wdyc0qc";
+  };
+
+  propagatedBuildInputs = [ aiofiles cryptography libusb1 pyasn1 rsa ];
+
+  # Tests aren't included in pypi package
+  doCheck = false;
+
+  meta = with lib; {
+    description =
+      "A Python implementation of ADB with shell and FileSync functionality.";
+    homepage = "https://github.com/JeffLIrion/adb_shell";
+    license = licenses.asl20;
```

Comment:
Add yourself as maintainer.

Reply:
Should that come as a separate PR, or as part of this PR?

Reply:
If you are not already listed as a maintainer, add yourself in a separate commit. Then reference yourself in the commit that adds the new package.

Reply:
@mweinelt Can you please review #98478 as well?

Reply:
Done, please rebase this branch.